src/microjson/utils.py

```diff
@@ -390,69 +396,74 @@ def polygons_to_microjson(  # noqa: PLR0915
             out_name = Path(self.out_dir, outname)
             with Path.open(out_name, "w") as f:
                 f.write(
-                    feature_collection.model_dump_json(indent=2, exclude_unset=True),
+                    feature_collection.model_dump_json(
+                        indent=2, exclude_unset=True),
                 )
                 logger.info(f"Saving overlay json file: {out_name}")
 
 
 class CustomValidation(pydantic.BaseModel):
     """Properties with validation."""
-    out_dir:Union[str, Path]
-    file_path:Union[str, Path]
+    out_dir: Union[str, Path]
+    file_path: Union[str, Path]
 
     @pydantic.validator("out_dir", pre=True)
     @classmethod
     def validate_out_dir(cls, value):
         if not Path(value).exists():
             raise ValueError(f"{value} do not exist! Please do check it again")
         if isinstance(value, str):
-            return Path(value) 
+            return Path(value)
         return value
+
     @pydantic.validator("file_path", pre=True)
     @classmethod
     def validate_file_path(cls, value):
         if not Path(value).exists():
-            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), value)
+            raise FileNotFoundError(
+                errno.ENOENT, os.strerror(errno.ENOENT), value)
         if not MicroJSON.model_validate(json.load(Path.open(Path(value)))):
             raise ValidationError(f"Not a valid MicroJSON {value.name}")
         if isinstance(value, str):
-            return Path(value) 
+            return Path(value)
         return value
-       
+
+
 class MicrojsonBinaryModel(CustomValidation):
     """Generate binary masks from polygons stored in microjson format.
 
     Args:
         out_dir: Path to output directory.
         file_path: Microjson file path
     """
-    out_dir:Union[str, Path]
-    file_path:Union[str, Path]
+    out_dir: Union[str, Path]
+    file_path: Union[str, Path]
 
     def microjson_to_binary(self) -> None:
-        """Convert polygon coordinates (series of points, rectangle) of all objects to binary mask"""
+        """Convert polygon coordinates (series of points, rectangle) of all
+        objects to binary mask"""
 
         data = json.load(Path.open(Path(self.file_path)))
         items = [Feature(**item) for item in data['features']]
         poly = [i.geometry.coordinates for i in items]
         meta = Properties(**data['properties'])
         image_name = meta.string.get("Image")
         x = int(meta.string.get("X"))
-        y= int(meta.string.get("Y"))
+        y = int(meta.string.get("Y"))
         fmask = np.zeros((x, y), dtype=np.uint8)
         for i, _ in enumerate(poly):
             image = fmask.copy()
             pol = np.array(poly[i][0])
             mask = sk.draw.polygon2mask((x, y), pol)
-            image[mask == False] = 0
-            image[mask == True] = 1
+            image[not mask] = 0
```

Comment:
Have you tested this change? I wonder may be this will break?

Reply:
I had some other problems with the test case. I will revert this change for now, I think you are right.